### DISCUSSION TITLE
How do you refresh the source files in the project dir for each platform?

### DISCUSSION THREAD

**Discussion by ItsCubeTime** [beeware/toga#1789] [General]
Hi 👋If I modify the files under root/src - how do I get those changes to push to my targets like root/android & root/windows?

I would have thought `briefcase build` would do this for me, but it doesnt?

**Comment by freakboy3742** [comment]
This is more of a Briefcase question than a Toga question; but `briefcase update` is what you're looking for. You can also request an update as part of the build by running `briefcase build -u`.

FWIW - this is covered in [step 4 of the BeeWare tutorial](https://docs.beeware.org/en/latest/tutorial/tutorial-4.html); if you haven't run through that tutorial already, you may want to do so, as it will cover a couple of other edge cases that are common sources of confusion.

**Reply by ItsCubeTime** [reply]
Thank you! ✨
